lib/services/analytics/analytics-service.ts

```diff
@@ -104,8 +104,8 @@ export class AnalyticsService extends AnalyticsServiceBase {
 		await this.trackInGoogleAnalytics(googleAnalyticsEventData);
 	}
 
-	public dispose(): void {
-		if (this.brokerProcess && this.shouldDisposeInstance) {
+	public dispose(disposeOptions?: IForceOption): void {
+		if ((disposeOptions && disposeOptions.force) || (this.brokerProcess && this.shouldDisposeInstance)) {
```

Comment:
In case you do not have a brokerProcess, but disposeOptions.force is true, you'll have an error on the next line.